tests/tests_pytorch/loops/test_loops.py

```diff
@@ -28,8 +28,11 @@
 from pytorch_lightning.loops import EvaluationLoop, Loop, TrainingBatchLoop, TrainingEpochLoop
 from pytorch_lightning.trainer.progress import BaseProgress
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from tests_pytorch.helpers.dataloaders import unpatch_dataloaders
 from tests_pytorch.helpers.runif import RunIf
 
+unpatch_dataloaders()
```

Comment:
What happens if you remove this?

Reply:
all tests that verify the patching fail, because

```
assert dataloader not patched  # fails here because already patched on import
with context_manager():
    assert dataloader patched
```

and similar.

Reply:
I would suggest putting it at the beginning of each test that requires it then